packages/cli/src/constructs/__tests__/multi-step-check.spec.ts

```
@@ -53,58 +58,78 @@ describe('MultistepCheck', () => {
-    delete Session.defaultRuntimeId
+    try {
+      Session.defaultRuntimeId = '9999.99'
+      const diagnostics = new Diagnostics()
+      const check = new MultiStepCheck('main-check', {
+        name: 'Main Check',
+        code: { content: '' },
+      })
+      await check.validate(diagnostics)
+      expect(diagnostics.isFatal()).toBe(true)
+    } finally {
+      delete Session.defaultRuntimeId
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)